cpp/oneapi/dal/algo/kmeans/backend/gpu/train_kernel_lloyd_csr_dpc.cpp

```diff
@@ -206,11 +223,6 @@ struct train_kernel_gpu<Float, method::lloyd_csr, task::clustering> {
             calc_objective_function(queue,
                                     arr_closest_distances,
                                     { last_event, centroid_squares_event, assign_event });
-        {
-            // Reduce objective function value over all ranks
-            auto obj_func_reduce_event = comm.allreduce(objective_function);
```

Comment:
I didnt find any replacement for comm usage, can you please clarify why it works in another way now